knowhere/index/VecIndex.h

```diff
@@ -43,11 +43,26 @@ class VecIndex : public Index {
     virtual void
     AddWithoutIds(const DatasetPtr& dataset, const Config& config) = 0;
 
+    /**
+     * @brief Prepare the Index ready for query.
+     * 
+     * @return true if the index is well prepared.
+     * @return false if any error.
+     */
+    virtual bool
+    Prepare() {
+        KNOWHERE_THROW_MSG("Prepare not supported yet");
+    }
+
     virtual DatasetPtr
     GetVectorById(const DatasetPtr& dataset, const Config& config) {
         KNOWHERE_THROW_MSG("GetVectorById not supported yet");
     }
 
+    /**
+     * @brief TopK Query. Will fill the return ids&distances with -1 if the result size is smaller than K.
```

Comment:
when no enough result, id will be set -1, distance will be set float_min(IP) or float_max(L2)